app/constants/request_date.py

```diff
@@ -0,0 +1 @@
+RELEASE_PUBLIC_DAYS = 20
```

Comment:
Newline added. 
I believe we are planning to add other constants to this file including as you stated, ACKNOWLEDGEMENT_DAYS_DUE.